packages/kbn-cell-actions/src/actions/filter/filter_out.ts

```diff
@@ -8,7 +8,9 @@
 import { i18n } from '@kbn/i18n';
 import type { FilterManager, KBN_FIELD_TYPES } from '@kbn/data-plugin/public';
 import { NotificationsStart } from '@kbn/core-notifications-browser';
-import { createFilter, isEmptyFilterValue } from './create_filter';
+import { isEmptyFilterValue } from './create_filter';
+import { addFilter } from './add_filter';
```

Comment:
NIT: It probably makes more sense now to move the `isEmptyFilterValue` function to the new _add_filter.ts_ module as well. So actions will only have to import from _add_filter_ module, and the _create_filter_ module will become an internal module to help create the filters of _add_filter_ module.